renderer/src/VulkanRenderer.hpp

```diff
@@ -119,13 +119,12 @@ class R_PUBLIC_API VulkanRenderer {
     bool createCommandPool(VkDevice logical_device);
     bool createCommandBuffers(VkDevice logical_device);
     bool createSyncObjects(VkDevice logical_device);
-    bool recreateSwapchain(VkPhysicalDevice device, VkDevice logical_device, bool handle_rendering_objects = false);
+    bool recreateSwapchain(VkPhysicalDevice device, VkDevice logical_device, bool objects_added = false, bool resized = false);
     void cleanupSwapchain(bool final_cleanup = true);
     bool createDescriptorSetLayout(VkDevice logical_device);
     VkSampleCountFlagBits getMaxUsableSamplesCount();
     bool createColorResources(VkDevice logical_device);
-    preexistant_objects_load_result loadPreexistentObjects(bool secondary_invokation = false);
-    bool initPreexistentTextures(preexistant_objects_load_result &objects);
+    preexistant_objects_load_result loadPreexistentObjects(bool secondary_invokation, bool added_object = false, bool resized = false);
```

Comment:
`preexistent_objects_load_result` or better yet, `predef_objs_load_result`. Same for the name `loadPreexistentObjects` -> `loadPredefObjs`